homeassistant/components/recorder/statistics.py

```diff
@@ -1505,7 +1505,7 @@ def statistic_during_period(
         if end_time is None:
             tail_start_time = _last_statistic(session, Statistics, metadata_id)
             if tail_start_time:
-                tail_start_time += timedelta(hours=1)
+                tail_start_time += Statistics.duration
```

Comment:
There are a few above as well, but I that better to adjust in another PR

Reply:
Yeah, I didn't want to touch those in a PR marked for a patch release.